frontend-next-migration/src/entities/Hero/ui/BarChart/BarChart.tsx

```diff
@@ -0,0 +1,160 @@
+import cls from './BarChart.module.scss';
+import { useClientTranslation } from '@/shared/i18n';
+
+export type Stats = {
+    [key: string]: number;
+};
+
+export type BarChartProps = {
+    stats: Stats;
+    width: number;
+    height: number;
+};
```

Comment:
may be instead 
``` 
export type Stats = {
    [key: string]: number;
};

export type BarChartProps = {
    stats: Stats;
    width: number;
    height: number;
};
```

use this approach: 

```
export type Stat = {
    name: string;
    value: number;
};

export type BarChartProps = {
    stats: Stat[];
    width: number;
    height: number;
};

``` 
# Why This is Better

- Readability: Stat[] allows for direct mapping without Object.keys() or Object.entries(), improving readability.
- Flexibility: This structure is more extensible, should additional properties be needed for each stat.

Reply:
and I would also add   the colour here : 
```
export type Stat = {
    name: string;
    value: number;
    color: string;
}; 
``` to avoid hardcoding in css